src/main/java/mil/dds/anet/beans/Person.java

```diff
@@ -224,6 +224,19 @@ public Position getPosition() {
         });
   }
 
+  @GraphQLQuery(name = "allPreviousPositions")
```

Comment:
It seems to me this accessor is only added for the server-side tests, not because it is needed for front-end functionality. That does not seem like a good idea to me.

Reply:
GraphQLQuery removed